PWGHF/TableProducer/HFTreeCreatorLbToLcPi.cxx

```diff
@@ -0,0 +1,361 @@
+                  full::NSigRICHTrk2Ka,
+                  full::NSigRICHTrk3Pi,
+                  full::NSigRICHTrk3Pr,
+                  full::NSigfRICHPi0,
+                  full::NSigfRICHTrk1Pi,
+                  full::NSigfRICHTrk1Pr,
+                  full::NSigfRICHTrk2Ka,
+                  full::NSigfRICHTrk3Pi,
+                  full::NSigfRICHTrk3Pr,
+                  full::NSigTOFTrk1Pi,
+                  full::NSigTOFTrk1Pr,
+                  full::NSigTOFTrk2Ka,
+                  full::NSigTOFTrk3Pi,
+                  full::NSigTOFTrk3Pr,
+                  full::LcM,
+                  full::LcCt,
+                  full::LcY,
+                  full::LcE,
+                  full::LcEta,
+                  full::LcCPA,
+                  full::LcCPAXY,
+                  full::LcChi2PCA,
+                  full::LcDecayLength,
+                  full::LcDecayLengthXY,
+                  full::LcDecayLengthNormalised,
+                  full::LcImpactParameter0,
+                  full::LcImpactParameter1,
+                  full::LcImpactParameter2,
+                  full::CandidateSelFlag,
+                  full::M,
+                  full::Pt,
+                  full::P,
+                  full::CPA,
+                  full::CPAXY,
+                  full::Ct,
+                  full::Eta,
+                  full::Phi,
+                  full::Y,
+                  full::MCflag);
+
+DECLARE_SOA_TABLE(HfCandLbFullEvents, "AOD", "HFCANDLbFullE",
+                  collision::BCId,
+                  collision::NumContrib,
+                  collision::PosX,
+                  collision::PosY,
+                  collision::PosZ,
+                  full::IsEventReject,
+                  full::RunNumber);
+
+DECLARE_SOA_TABLE(HfCandLbFullParticles, "AOD", "HFCANDLbFullP",
+                  collision::BCId,
+                  full::Pt,
+                  full::Eta,
+                  full::Phi,
+                  full::Y,
+                  full::MCflag);
+
+} // namespace o2::aod
+
+namespace o2::aod
+{
+namespace hf_track_index_alice3_pid
+{
+DECLARE_SOA_INDEX_COLUMN(Track, track); //!
+DECLARE_SOA_INDEX_COLUMN(RICH, rich);   //!
+DECLARE_SOA_INDEX_COLUMN(FRICH, frich); //!
+} // namespace hf_track_index_alice3_pid
+
+DECLARE_SOA_INDEX_TABLE_USER(HfTrackIndexALICE3PID, Tracks, "HFTRKIDXA3PID", //!
+                             hf_track_index_alice3_pid::TrackId,
+                             hf_track_index_alice3_pid::RICHId,
+                             hf_track_index_alice3_pid::FRICHId);
+} // namespace o2::aod
+
+struct Alice3PidIndexBuilder {
+  Builds<o2::aod::HfTrackIndexALICE3PID> index;
+  void init(o2::framework::InitContext&) {}
+};
+
+/// Writes the full information in an output TTree
+struct HfTreeCreatorLbToLcPi {
+  Produces<o2::aod::HfCandLbFull> rowCandidateFull;
+  Produces<o2::aod::HfCandLbFullEvents> rowCandidateFullEvents;
+  Produces<o2::aod::HfCandLbFullParticles> rowCandidateFullParticles;
```

Comment:
You are not filling these tables.

Reply:
The table are still there.

Reply:
That is odd, when I attempt to change the file, these lines are already gone. In the branch the commit was pushed already.
eg: https://github.com/mjongerh/O2Physics/blob/Lb_PRbranch/PWGHF/TableProducer/HFTreeCreatorLbToLcPi.cxx#L211
https://github.com/AliceO2Group/O2Physics/pull/994/commits/11fbeb3e8155feb833d3441b7e2c22ea5a80c6fd

Reply:
What I meant is that you just removed the two lines but not the definitions of columns and tables.